controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -197,11 +601,11 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           };
         }
 
-        if (await fedGraphRepo.exists(req.name)) {
+        if (await fedGraphRepo.byName(req.name, req.namespace)) {
```

Comment:
I saw that we ensure the default namespace in the CLI. This is not backwards compatible, because we can't assume people upgrade immediately to the latest CLI version. We have to ensure a default namespace on the controlplane.

Reply:
Moved the default to CP